README.md

```diff
@@ -1,68 +1,24 @@
-I generate speech samples based on [Harvard Sentences](http://www.cs.columbia.edu/~hgs/audio/harvard.html) as the original paper does. It is already included in the repo.
-
-  * Run `synthesize.py` and check the files in `samples`.
-
-## Generated Samples
-
-| Dataset       | Samples |
-| :----- |:-------------|
-| LJ      | [50k](https://soundcloud.com/kyubyong-park/sets/dc_tts) [200k](https://soundcloud.com/kyubyong-park/sets/dc_tts_lj_200k) [310k](https://soundcloud.com/kyubyong-park/sets/dc_tts_lj_310k) [800k](https://soundcloud.com/kyubyong-park/sets/dc_tts_lj_800k)|
-| Nick      | [40k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_40k) [170k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_170k) [300k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_300k) [800k](https://soundcloud.com/kyubyong-park/sets/dc_tts_nick_800k)|
-| Kate| [40k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_40k) [160k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_160k) [300k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_300k) [800k](https://soundcloud.com/kyubyong-park/sets/dc_tts_kate_800k) |
-| KSS| [400k](https://soundcloud.com/kyubyong-park/sets/dc_tts_ko_400k) |
-
-## Pretrained Model for LJ
-
-Download [this](https://www.dropbox.com/s/1oyipstjxh2n5wo/LJ_logdir.tar?dl=0).
-
-## Notes
-
-  * The paper didn't mention normalization, but without normalization I couldn't get it to work. So I added layer normalization.
-  * The paper fixed the learning rate to 0.001, but it didn't work for me. So I decayed it.
-  * I tried to train Text2Mel and SSRN simultaneously, but it didn't work. I guess separating those two networks mitigates the burden of training.
-  * The authors claimed that the model can be trained within a day, but unfortunately the luck was not mine. However obviously this is much fater than Tacotron as it uses only convolution layers.
-  * Thanks to the guided attention, the attention plot looks monotonic almost from the beginning. I guess this seems to hold the aligment tight so it won't lose track.
-  * The paper didn't mention dropouts. I applied them as I believe it helps for regularization.
-  * Check also other TTS models such as [Tacotron](https://github.com/kyubyong/tacotron) and [Deep Voice 3](https://github.com/kyubyong/deepvoice3).
-  
+## Requirements
+  * NumPy >= 1.11.1
+  * TensorFlow >= 1.3 (Note that the API of `tf.contrib.layers.layer_norm` has changed since 1.3)
+  * librosa
+  * tqdm
+  * matplotlib
+  * scipy
+  
+## Data
+we are using a pretrained model from https://github.com/Kyubyong/dc_tts , which is trained by LJ Speech Dataset. The seven wav files in samples folder are the synthesized audio.
+
+## Synthesis
+We generate speech samples based on our programing course description. Its txt file is included in the repo.
+  * Download the pretrained model and unzip it. Then create a new file named "logdir" in the directory, and put the two model folders inside of it.
```

Comment:
Maybe discuss some of your findings & results at the end of the readme file?

Reply:
We did not consider this to be necessarily part of a repo but we added some things that we noticed in a "findings" section in the readme file.

Reply:
thanks for adding that!